docs/index.css

```diff
@@ -99,3 +99,24 @@ code {
 .Home__links {
   margin: 2px 0 0 3px;
 }
+
+@media (max-width: 590px) {
+  .SideBar--with-footer,
+  .Layout {
+    bottom: 4.55rem;
+  }
+}
+
+@media (max-width: 370px) {
+  .SideBar--with-footer,
+  .Layout {
+    bottom: 5.7rem;
+  }
+}
+
+@media (max-width: 300px) {
+  .SideBar--with-footer,
+  .Layout {
+    bottom: 6.8rem;
+  }
+}
```

Comment:
What do you think about using a mobile-first approach and using `min-width` for each media query as the screen size increases?

Using `max-width` media queries is more desktop-first.

> A mobile-first can usually be spotted by it’s use of min-width instead of max-width. If we start with min-width, we’re essentially saying, “hey, browser, start here and work up.” On the flip side, max-width is sort of like prioritizing larger screens.

https://css-tricks.com/a-complete-guide-to-css-media-queries/

I tried this out and I think it works alright. This also removes one extra media query.

```
.SideBar--with-footer,
.Layout {
  bottom: 5.72rem;
}

@media (min-width: 371px) {
  .SideBar--with-footer,
  .Layout {
    bottom: 4.62625rem;
  }
}

@media (min-width: 591px) {
  .SideBar--with-footer,
  .Layout {
    bottom: 3.5325rem;
  }
}
```

https://user-images.githubusercontent.com/3081483/190518400-9a0c77b9-2b8e-4a10-acf2-57c1aba85018.mov

Reply:
You right. You right. I should have done mobile-first.

I'm keeping below 300px because I'm that way and design mentioned responsive down to like 250px when I was working on a different project, though.

Reply:
Gotta watch out for those people on their apple watches using the pattern library site!

Reply:
I'm here for our developers on the go.